core/views/forms.py

```diff
@@ -157,7 +157,7 @@ def __init__(self, *arg, **kwargs):
                 else:
                     user = User.objects.filter(username=data["username"]).first()
                 data["username"] = user.username
-            except:
+            except:  # noqa E722 I don't what error is supposed to be raised here
```

Comment:
`           except:  # noqa E722 I don't know what error is supposed to be raised here`

Reply:
De fait, oui, en lisant le code du try, j'ai aucune idée de ce qui est censé foirer ici.

Reply:
bv